src/components/dev-hub/academia-sign-up-form.js

```diff
@@ -0,0 +1,204 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import { colorMap, size, screenSize } from './theme';
+import { P, ArticleH3 } from './text';
+import Input from './input';
+import Button from './button';
+import Select from './select';
+import Checkbox from '@leafygreen-ui/checkbox';
+import SectionHeader from './section-header';
+import SuccessState from './success-state';
+
+const INPUT_BOX_WIDTH = '335px';
+
+const StyledButton = styled(Button)`
+    display: flex;
+    margin: ${size.large} auto;
+`;
+
+const StyledSelect = styled(Select)`
+    color: ${colorMap.greyLightTwo};
+`;
+
+const ErrorMessage = styled(P)`
+    color: ${colorMap.salmon};
+`;
+
+const StyledSectionText = styled(ArticleH3)`
+    color: ${colorMap.greyLightTwo};
+    font-family: akzidenz;
+    font-weight: normal;
+`;
+
+const InstructorSection = styled('div')`
+    display: grid;
+    grid-template-columns: repeat(auto-fill, ${INPUT_BOX_WIDTH});
+    grid-row-gap: ${size.mediumLarge};
+    justify-content: space-between;
+    margin-bottom: ${size.xlarge};
+    @media ${screenSize.upToMedium} {
+        display: grid;
+        grid-template-columns: repeat(auto-fill, ${INPUT_BOX_WIDTH});
+        grid-row-gap: ${size.mediumLarge};
+        justify-content: space-between;
+        margin-bottom: ${size.xlarge};
+    }
+`;
+
+const InstitutionSection = styled('div')`
+    display: grid;
+    grid-template-columns: repeat(auto-fill, ${INPUT_BOX_WIDTH});
+    grid-row-gap: ${size.mediumLarge};
+    justify-content: space-between;
+    margin-bottom: ${size.xlarge};
+    @media ${screenSize.upToMedium} {
```

Comment:
Looks like here and in `InstructorSection` we have css used both in and out of the media query. These styles will carry over so we can omit whatever is the same (such as `display: grid`, `grid-row-gap`, `justify-content` and `margin-bottom` all look the same so let's clean that up!)